x-pack/plugins/stack_alerts/public/alert_types/geo_threshold/query_builder/expressions/entity_index_expression.tsx

```diff
@@ -150,7 +150,7 @@ export const EntityIndexExpression: FunctionComponent<Props> = ({
     <ExpressionWithPopover
       isInvalid={isInvalid}
       value={indexPattern.title}
-      defaultValue={'Select an index pattern and geo shape/point field'}
+      defaultValue={'Select an index pattern and geo point field'}
```

Comment:
this should be i18n message. Plus how about moving to common/i18n file to avoid duplicate message

Reply:
Good catch. I've made the i18n updates but kept them inline rather than moving to a common file to be consistent with how other alerts appear to do it. Also, these 2 alerts are likely to consolidate in the near future